src/vignette/storage/static_assets.clj

```diff
@@ -0,0 +1,48 @@
+(ns vignette.storage.static-assets
+  (:require [vignette.storage.protocols :refer :all]
+            [org.httpkit.client :as http]
```

Comment:
Jet client only supports pure async interface. While http-kit client provides the fallback to streaming interface as a thin wrapper on async callbacks. 
The idea is to compare the performance once jet client implements the streaming wrapper.
Alternatively the vignette might be refactored to core.async flow which would make jet client usable.